### PR TITLE
Check allow-transfer in custom option when the zone is slave

### PR DESCRIPTION
**Pull request by Am1g0B0y**
If i add custom option (allow-transfer) to my slave zone, bind exit with error, because say already defined this option.
Created this modification and the zone finally good for zone transfer (if you want to enable zone tranfer for your slave zone).
Please accept this modification or enable allow-transfer acl selection in the GUI.

https://redmine.pfsense.org/issues/9916

### CODE REVIEW

**Comment by jim-p** [comment]
To effectively track changes and bug/feature requests, we require an entry on the [pfSense Redmine issue tracker](https://redmine.pfsense.org) associated with every pull request, and likewise the Redmine entry should also have a link back to the pull request.

Please associate this PR to a Redmine issue either by locating an existing issue at https://redmine.pfsense.org or by creating a new issue. Add a link to the Redmine issue that points to this PR, and also add a link on the PR to the Redmine issue.

For more information, see https://www.netgate.com/docs/pfsense/development/submitting-a-pull-request-via-github.html

Thanks!

**Comment by Am1g0B0y** [comment]
Sorry, redmine issue created with link here, first comment updated with link to issue: https://redmine.pfsense.org/issues/9916

File: dns/pfSense-pkg-bind/files/usr/local/pkg/bind.inc [hunk 1]
```diff
@@ -450,7 +450,10 @@ EOD;
 					case 'slave':
 						$bind_conf .= "\t\tmasters { $zoneipslave; };\n";
 						$bind_conf .= "\t\tallow-query { $zoneallowquery; };\n";
-						$bind_conf .= "\t\tallow-transfer { $zoneallowtransfer; };\n";
+						//$bind_conf .= "\t\tallow-transfer { $zoneallowtransfer; };\n";
```
Comment (rbgarga) [inline dns/pfSense-pkg-bind/files/usr/local/pkg/bind.inc]: Please remove commented out line

File: dns/pfSense-pkg-bind/files/usr/local/pkg/bind.inc [hunk 2]
```diff
@@ -450,7 +450,10 @@ EOD;
...
+							if ( $zonecustom == '' or ($zonecustom != '' and !preg_match('/allow-transfer/', $zonecustom))) {
```
Comment (rbgarga) [inline dns/pfSense-pkg-bind/files/usr/local/pkg/bind.inc]: You don't need to test if `$zonecustom` is empty or not, just call preg_match is enough here. Please also fix indentation, it's one level ahead previous lines

Comment (Am1g0B0y) [reply dns/pfSense-pkg-bind/files/usr/local/pkg/bind.inc]: You are right :)
